scripts/var_props.py

```diff
@@ -881,7 +883,9 @@ def __init__(self, var1_stdname, var1_type, var1_kind, var1_units,
         self.has_vert_transforms = False
         incompat_reason = list()
         # First, check for fatal incompatibilities
-        if var1_stdname != var2_stdname:
+        # If it's a tendency variable, it's assumed the standard name is of the
+        #  form "tendency_of_var2_stdname"
+        if not is_tend and  var1_stdname != var2_stdname:
```

Comment:
Should we add a check that <var1> starts with `tendency_ `when `is_tend=True`? Or is this handled somewhere else? If redundant, ignore.

Reply:
good call! as it's currently used, it's only called when a variable starts with "tendency_of" but I added the check to prevent future (by me) implementation issues!